src/main/java/com/ericsson/ei/services/SubscriptionService.java

```diff
@@ -142,8 +144,10 @@ public boolean deleteSubscription(String subscriptionName) throws AccessExceptio
         if (deleteResult) {
             String subscriptionIdQuery = String.format(SUBSCRIPTION_ID, subscriptionName);
             if (!cleanSubscriptionRepeatFlagHandlerDb(subscriptionIdQuery)) {
-                LOG.error("Failed to clean subscription \"" + subscriptionName
-                        + "\" matched AggregatedObjIds from RepeatFlagHandler database");
+                LOG.warn("Subscription  \"" + subscriptionName
+                        + "\" matched AggregatedObjIds from RepeatFlagHandler database could not be cleaned during the removal of subscription,\n"
+                		+ "probably due to subscritpiton has never matched any Aggregated Objects and "
+                        + "no matched aggredgatedObjectsIds has been stored in database for the specific subscription.");
```

Comment:
Same spelling mistakes in this log message.

Reply:
Done.